lib/auth/tls_test.go

```diff
@@ -2914,26 +2914,6 @@ func (s *TLSSuite) TestEventsClusterConfig(c *check.C) {
 	suite.ExpectResource(c, w, 3*time.Second, clusterNameResource)
 }
 
-func (s *TLSSuite) TestAPIBackwardsCompatibilityLogic(c *check.C) {
```

Comment:
DeleteUser, getUser, and getUsers were introduced in 4.3 or earlier, and no longer need to be backwards compatible.